integrations/README.md

```diff
@@ -12,4 +12,6 @@ This folder contains the integrations for OpenDAL. Integrations are used to inte
 
 - [`virtiofs_opendal`](virtiofs/README.md): Use OpenDAL as a backend to access data in various services with VirtioFS protocol.
 
-- [`unftp-sbe-opendal`](unftp-sbe/README.md): Use OpenDAL as a backend to access data in various services with FTP protocol.
\ No newline at end of file
+- [`unftp-sbe-opendal`](unftp-sbe/README.md): Use OpenDAL as a backend to access data in various services with FTP protocol.
+
+- [`cloudfilter_opendal`](cloudfilter/README.md): Use OpenDAL as a backend to access data in various services with CloudFilter.
```

Comment:
Would you like to provide a link to `cloudfilter` and provide some description about this crate?